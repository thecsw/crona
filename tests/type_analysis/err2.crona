foo : void() {}

main : void() {
    write foo();
}