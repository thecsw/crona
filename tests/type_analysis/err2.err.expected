FATAL [4,11]: Attempt to output void
Type Analysis Failed
